setup.cfg

```diff
@@ -13,4 +13,4 @@ license_file = LICENSE.txt
 
 [flake8]
 max-line-length = 88
-ignore = W503, E121, E126, E211, E225, E501, E203, E402, F401, F811, F821
+ignore = W503, E121, E126, E211, E225, E501, E203, E402, F401, F811
```

Comment:
Why this change?

Reply:
That's so that we get warnings for using variables that are not defined, not sure why it was disabled since it's super useful.